apis/external/v1alpha1/constants.go

```diff
@@ -39,12 +39,12 @@ const (
 	ExternalDNSPhaseInProgress ExternalDNSPhase = "InProgress"
 )
 
-// kubebuilder:validation:Enum:=aws;cloudflare
```

Comment:
Why remove it?

Reply:
If we add new provider we have to add the provider enum here also. I handled the error 'invalid provider' internally.

Reply:
okey